test/DSRMainnetOracle.t.sol

```diff
@@ -0,0 +1,238 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "forge-std/Test.sol";
+
+import { PotMock } from "./mocks/PotMock.sol";
+
+import { DSRMainnetOracle, IDSROracle } from "../src/DSRMainnetOracle.sol";
+
+contract DSRMainnetOracleTest is Test {
+
+    event SetPotData(IDSROracle.PotData nextData);
+
+    uint256 constant FIVE_PCT_APY_DSR        = 1.000000001547125957863212448e27;
+    uint256 constant FIVE_PCT_APY_APR        = 0.048790164207174267760128000e27;
+    uint256 constant ONE_HUNDRED_PCT_APY_DSR = 1.00000002197955315123915302e27;
+
+    uint256 ONE_YEAR;
+
+    PotMock          pot;
+    DSRMainnetOracle oracle;
+
+    function setUp() public {
+        // To get some reasonable timestamps that are not 1
+        skip(1 * (365 days));
+
+        pot = new PotMock();
+
+        skip(1 * (365 days));
+
+        ONE_YEAR = block.timestamp + 365 days;
+
+        oracle = new DSRMainnetOracle(address(pot));
+    }
+
+    function test_storage_defaults() public {
+        assertEq(oracle.getDSR(), pot.dsr());
+        assertEq(oracle.getChi(), pot.chi());
+        assertEq(oracle.getRho(), pot.rho());
```

Comment:
Lets add three more assertions with explicit values just in case these are all zero

Reply:
fixed